### ISSUE TITLE
ENHANCEMENT - idle weapon finder

### ISSUE THREAD

**Issue by evanbowman** [evanbowman/skyland-beta#104]
**Describe wanted behaviour**
Create a button shortcut for jumping the cursor to a weapon that is either idle, or all of its targets have been destroyed.
But a lot of button combos are already in use...
Alternative: show an icon overtop of weapons that need new instructions

**Comment by Doronimmo** [comment]
Maybe the idle weapon could show in the minimap

**Comment by evanbowman** [comment]
> Maybe the idle weapon could show in the minimap

Good idea! Soon I need to work on optimizing the minimap, it was only originally intended to display during weapon selection, and it will affect frame rate even more if I keep adding new stuff to it.
